index.html

```diff
@@ -661,24 +734,6 @@
                     mxEvent.consume(evt);
                 };
 
-                // Returns a shorter label if the cell is collapsed and no
-                // label for expanded groups
-                graph.getLabel = function(cell) {
```

Comment:
Try double-clicking on the production server. It fails. In fact nothing works after the action. This function was directly related to it.